packages/qwik/src/core/shared/shared-container.ts

```diff
@@ -48,6 +49,10 @@ export abstract class _SharedContainer implements Container {
     return trackSignalAndAssignHost(signal, subscriber, property, this, data);
   }
 
+  serializeAttributeValue(key: string, value: any, styleScopedId?: string | null) {
```

Comment:
Why not simply re-export?

Reply:
yes, I forgot about reverting that change